Bruin Bite/Network/UserManager.swift

```diff
@@ -172,6 +172,19 @@ class UserManager {
         }
     }
 
+    func invalidateAccessToken(token: String) {
```

Comment:
rename this function to `logOutUser`and just copy the two lines from the current `logOutUser()` into this function.

Reply:
Gotcha